new-lamassu-admin/src/utils/constants.js

```diff
@@ -9,6 +9,11 @@ const MANUAL = 'manual'
 const IP_CHECK_REGEX = /^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$/
 
 const SWEEPABLE_CRYPTOS = ['ETH']
+const WARNING_LEVELS = {
+  CLEAN: 'clean',
+  PARTIAL: 'partial',
+  IMPORTANT: 'important'
+}
```

Comment:
Looks like this is pretty specific of the Services page. Any reason why is on the generic constant file?